servers/rendering/renderer_scene_cull.cpp

```diff
@@ -191,22 +191,22 @@ void RendererSceneCull::_instance_pair(Instance *p_A, Instance *p_B) {
 
 		if (A->scenario && A->array_index >= 0) {
 			InstanceData &idata = A->scenario->instance_data[A->array_index];
-			idata.flags |= InstanceData::FLAG_GEOM_LIGHTING_DIRTY;
+			idata.flags |= uint32_t(InstanceData::FLAG_GEOM_LIGHTING_DIRTY);
```

Comment:
I think we should define the Instancedata::Flags enum with a size of uint32_t instead of casting. It looks like we could basically remove all the changes in this file by doing:

`enum Flags : uint32_t {`